pkgs/applications/version-management/gitlab/default.nix

```diff
@@ -32,6 +32,10 @@ let
         openssl = x.openssl // {
           buildInputs = [ openssl ];
         };
+        ruby-magic-static = x.ruby-magic-static // {
+          buildInputs = [ file ];
+          buildFlags = [ "--enable-system-libraries" ];
+        };
```

Comment:
Maybe we can move those override into the global bundle override so that everyone benefits from it.

Reply:
They might be unnecessary in the next version, if they change back to mimemagic.

Reply:
> if they're needed everywhere

if a package in the default upstream configuration needs extra build time packages, options or flags then they should be in the global file which you can find at https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/ruby-modules/gem-config/default.nix.

It makes it easier to use gems in nixpkgs and each package or software does not need to repeat the same overrides.

> we should already have the necessary overrides to build openssl bindings in the global gemconfig?

As far as I can tell they are already in the global config and shouldn't be required here https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/ruby-modules/gem-config/default.nix#L430